bundles/AdminBundle/Controller/Admin/LoginController.php

```diff
@@ -229,6 +248,12 @@ public function lostpasswordAction(Request $request, ?BruteforceProtectionHandle
 
         $csrfProtection->regenerateCsrfToken();
 
+        if ($error){
+            $params['reset_error'] = 'There is no valid account with the info you provided. Please make sure you are entering a correct input.';
```

Comment:
```suggestion
            $params['reset_error'] = 'It was not possible to reset password with the info you provided. Please make sure you are entering a correct input.';
```

Reply:
maybe get rid of the line `It was not possible to reset password with the info you provided.` as the error starts with `An error occured when resetting password`